### ISSUE TITLE
Cannot connect to ZM server that does not have authentication setup

### ISSUE THREAD

**Issue by themoosman** [ZoneMinder/pyzm#7]
When calling `zapi = zmapi.ZMApi(options=cfg.api_options)` with `api_options` that don't include a user id and password the following stack it thrown.  My ZM server does not require authentication.
```
  File "/home/moos/git/house-monitor/status.py", line 68, in zm
    zapi = zmapi.ZMApi(options=cfg.api_options)
  File "/home/moos/.local/lib/python3.8/site-packages/pyzm/api.py", line 58, in __init__
    self._login()
  File "/home/moos/.local/lib/python3.8/site-packages/pyzm/api.py", line 134, in _login
    data={'user': self.options['user'],
KeyError: 'user'
```

**Comment by pliablepixels** [comment]
Should be fixed in 0.1.12

**Comment by themoosman** [comment]
Yep, looks good.  Thank you, sir!
```
[INFO] Using simple log output (default)
[DEBUG 1] Guessing portal URL is: https://zm.foobar.red/zm
[DEBUG 1] Not using auth
06-13-2020 08:31:39 DEBUG    Starting new HTTPS connection (1): zm.foobar.red:443
06-13-2020 08:31:39 DEBUG    https://zm.foobar.red:443 "POST /zm/api/host/getVersion.json HTTP/1.1" 200 40
```
